src/GithubPlugin/Strings/en-US/Resources.resw

```diff
@@ -394,4 +394,8 @@
     <value>GitHub</value>
     <comment>Shown in various places, is GitHub even localized as a product name?</comment>
   </data>
+  <data name="Widget_Template.Query" xml:space="preserve">
+    <value>Query</value>
+    <comment>Shown in Widget</comment>

```

Comment:
I think this could be more descriptive, something like "Label for the query used to filter GitHub issues".

Reply:
Done